src/sdf/src/sdf.cpp

```diff
@@ -0,0 +1,357 @@
+// Copyright 2022 The Manifold Authors.
```

Comment:
2023

Reply:
I think we are missing copyright header in the cbind files, I will add them later as well